src/app/views/query-response/pivot-items/pivot-items.tsx

```diff
@@ -59,6 +63,40 @@ export const getPivotItems = (properties: any) => {
     );
   }
 
+  function displayResultComponent() {
```

Comment:
seems like we're calling this every time we resize so it may be causing the weirdness I see when I have content loaded in the response and try to resize.